src/core/storage/providers/local/local_file_entity.ts

```diff
@@ -1,25 +1,47 @@
+import axios from "axios";
 import { FileEntity } from "../../abstract/file_entity";
 
 export class LocalFileEntity implements FileEntity{
 
     fileName: string;
     mimeType: string;
     filePath: string;
+    rootDownloadPath : string = "http://localhost:8080/files/download/";
```

Comment:
any particular reason to use _rootDownloadPath_ and _rootUploadPath_ as opposed  to just _downloadPath_ or _uploadPath_